arena.h

```diff
@@ -375,6 +376,19 @@ char *arena_sprintf(Arena *a, const char *format, ...)
 
     return result;
 }
+
+char *arena_vsprintf(Arena *a, const char *format, va_list args)
+{
+    va_list args_copy;
+    va_copy(args_copy, args);
```

Comment:
If I understood correctly the original vsprintf from libc doesn't really copy the va_list:

![2025-04-21-010248_1050x248_scrot](https://github.com/user-attachments/assets/4ef93481-221f-46d6-9f26-85033caa0ded)

*from man 3 printf*

Maybe it makes sense to do the same for consistency?

Reply:
Ah, I think I just misunderstood your code sorry. It's ok.